SPECS/kernel-azure/config_aarch64

```diff
@@ -7392,10 +7390,8 @@ CONFIG_MMC_BCM2835=m
 CONFIG_MMC_MTK=m
 CONFIG_MMC_SDHCI_BRCMSTB=m
 CONFIG_MMC_SDHCI_XENON=m
-CONFIG_MMC_SDHCI_OMAP=m
```

Comment:
Changed due to https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/106136f2099b43b04f33787b53104621d73c1791

Reply:
A little more color into this. The check is failing because  `MMC_SDHCI_OMAP `got changed.  
 
In Mariner it was originally turned on in a huge batch here: [Enable arm64 hyperv and SoCs support for CBL-Mariner (#366) · microsoft/CBL-Mariner@d42ad21 (github.com)](https://github.com/microsoft/CBL-Mariner/commit/d42ad2134f8df440a484afe79f8195547123f2f5)
 
For this PR update,to 5.15.148.1, the config was changed to have depends on certain arches ([commit](https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/106136f2099b43b04f33787b53104621d73c1791))
`depends on ARCH_OMAP2PLUS || ARCH_KEYSTONE || COMPILE_TEST`
 
`ARCH_KEYSTONE `- [Kconfig](https://github.com/microsoft/CBL-Mariner-Linux-Kernel/blob/rolling-lts/mariner-2/5.15.148.1/arch/arm/mach-keystone/Kconfig)
`ARCH_OMAP2PLUS`- [Kconfig](https://github.com/microsoft/CBL-Mariner-Linux-Kernel/blob/rolling-lts/mariner-2/5.15.148.1/arch/arm/mach-omap2/Kconfig#L97) 
 
However, we never supported these arches in 2.0 so this config should imo go away and is not worth recording